tests/rustdoc/attributes.rs

```diff
@@ -11,3 +11,11 @@ pub extern "C" fn g() {}
 // @has foo/struct.Repr.html '//pre[@class="rust item-decl"]' '#[repr(C, align(8))]'
 #[repr(C, align(8))]
 pub struct Repr;
+
```

Comment:
Could you leave a short comment here explaining what this is checking (e.g., “checking for the presence of a chip that …”). It's not entirely obvious right now for people who don't know about this visual feature. Thanks!

Reply:
done